pkg/pillar/rootfs/blink.sh

```diff
@@ -6,19 +6,19 @@ repeat=-1
 
 if [ $# != 0 ]; then
```

Comment:
please let's delete blink.sh instead of fixing it. (that was a previous comment of mine that appears to have gotten lost)

Reply:
Sorry, I think I've missed. I will delete.